components/VM_Arm/configurations/vm.h

```diff
@@ -132,6 +132,7 @@
     time_server.simple = true;
 
 #define PER_VM_VIRTUAL_SERIAL_CONFIGURATION_DEF(num) \
+    vm##num.serial_getchar_attributes = num; \
```

Comment:
This same approach could be used for the output channels to ensure that the guests are always assigned the same colors?

Reply:
It could, yeah. But I haven't seen that behavior like I was seeing with the `getchar`.